src/index.js

```diff
@@ -1,7 +1,16 @@
 import low from 'lowlight'
 import visit from 'unist-util-visit'
 
-export default function attacher({include, exclude, prefix} = {}) {
+export default function attacher({include, exclude, prefix, languages} = {}) {
+
+  // register custom languages
+  if (languages) {
```

Comment:
it looks like languages is intended to be a plain object with language name as keys and a language function as the value?
Would it make sense to check that `languages` is a plain object here? maybe using https://github.com/jonschlinkert/is-plain-object

Reply:
I’d say it’s fine to assume folks pass sensical data 🤷‍♂️